.github/workflows/translate.yml

```diff
@@ -0,0 +1,71 @@
+name: Update Translations
+on:
+  push:
+    branches:
+      main
+
+# Cancel active CI runs for a PR before starting another run
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash  # https://github.com/beeware/briefcase/pull/912
+
+env:
+  FORCE_COLOR: "1"
+
+jobs:
+  update-translations:
```

Comment:
@rmartin16 The one problem I'm aware of with this workflow is that every merge to main that updates a document will cause 2 workflows to run - the first to update the translations, which will cause a commit to main, which will cause the translation workflow to run. The second workflow will identify that there haven't been any translation changes, so there shouldn't be a cascade... but it would be nice to avoid the second action run if possible. However, I can't think of any condition to apply to avoid the second run. Any thoughts?

Reply:
I wasn't sure if the actor would still be the same as it would be on a PR. The second pass should alwasy be a no-op (as in, there shouldn't be a commit cascade); but a shortcut will always be preferable. I'll do some poking around and see if the actor trick will work.